src/helpers/shared/helpers.js

```diff
@@ -63,3 +65,7 @@ export const sanitizeValues = (values, entityType, store) => {
   {});
   return udefinedToNull(permittedValues, nullableMapper[entityType]);
 };
+
+export const timeAgo = (date) => DateFormat ?
+  <span><DateFormat date={ date } type="relative"/></span>
+  :  moment(date).fromNow();
```

Comment:
Is this fallback required? DateFormat is always accessible if you import it above. You might have to update the version of `@redhat-cloud-services/frontend-components` if the component is not present.